signac_dashboard/templates/layout.html

```diff
@@ -99,9 +99,17 @@ <h2 class="subtitle is-4">Modules</h2>
                         </a>
                         {% endif %}
                         {% endblock module_selector %}
-                        {% endblock sidebar %}
-                        {% endif %}
                     </div>
+                    {% block logout_link %}
+                    <div class="menu is-fixed-bottom">
```

Comment:
I added a logout link to the sidebar, but I need help getting it to appear at the bottom of the sidebar instead of underneath the view and module selection boxes.